### PR TITLE
Fhac 1362-Remove unused dependency without using packageExclude

### PR DESCRIPTION
**Pull request by mhpnguyen**
Since there is no global exclusion dependencies in maven 3.x,  we can eliminate unused jar by using provided scope.  It also helps dependency check plugin to filter provided scope jars since they are not inside the binary ear.  
@TabassumJafri, @sjarral112: can you please review?

### CODE REVIEW

**Review by sjarral112 — COMMENTED** [review]
Please remove line 93-94.. redundant..

**Comment by christophermay07** [comment]
Are the global whitespace changes intended?

File: Product/Production/Deploy/jboss7/pom.xml [hunk 1]
```diff
@@ -88,22 +88,7 @@
...
-                        lib/geronimo*,
-                        lib/javax.servlet-api*,lib/json*,lib/jul*,lib/jsr311*,lib/jsr250*,lib/jackson*,
-                        lib/jandex*,lib/javassist-3.4*,lib/jcip*,lib/jsr173*,
-                        lib/not-yet*,
-                        lib/servlet-api*,lib/stax-api*,lib/spring-webmvc*,lib/spring-jms*,lib/spring-messaging*,
-                        lib/velocity*,
-                        lib/xml-apis*,lib/xmlbeans*,lib/xmlpull*,lib/xmltooling*,lib/xpp3*,lib/xstream*
+                        lib/Properties-*.jar
                     </packagingExcludes>
                     <!-- Explode all archives on server (extremely time-consuming) -->
```
Comment (sjarral112) [inline Product/Production/Deploy/jboss7/pom.xml]: Please remove line 93-94 ... redundant comments

**Comment by TabassumJafri** [comment]
jboss, weblogic and WAS pom.xml each have an entry of lib/Properties-*.jar in packageExclude. Is there a way to have this removed from each pom?

**Review by TabassumJafri — APPROVED** [review]
approved for merge

**Review by sjarral112 — APPROVED** [review]
Endorsed to merge

**Pull request merged**